bouture.js

```diff
@@ -59,30 +60,42 @@ function completeElement (tag, args) {
         Object.keys(arg)
           .forEach(attributeName => {
             const attributeValue = arg[attributeName]
-            switch (typeof attributeValue) {
-              case 'boolean':
-                if (attributeValue) {
-                  element.setAttribute(attributeName, '')
+            // check for event binding
+            const eventName = attributeName.match(/^on/) && attributeName.replace(/^once|on/, '')
+            if (eventNames.has(eventName)) {
```

Comment:
This means that the attribute must be passed as `onclick`.
The proposition in the readme was `onClick` (uppercase 'C'). I had chosen this form because [it's the one used in React](https://reactjs.org/docs/events.html#mouse-events).
One benefit i see with this form is that events stand out clearly in attribute objects